autodevops/action.yml

```diff
@@ -0,0 +1,42 @@
+name: "Autodevops"
+description: "SocialGouv automated deployment"
+inputs:
+  project:
+    description: "The project name, usually the repository name"
+  imageName:
+    description: "The image registry path"
+  dockerfile:
+    description: "The docker file path"
+  token:
+    description: "The GHCR authentication token"
+  kubeconfig:
+    description: "The Kubernetes authentication config"
+  environment:
+    description: "The deployment environment (dev | preprod | prod)"
+  productionNamespace:
+    description: "Override production namesapce"
+
+runs:
+  using: "composite"
+  steps:
+
+  - name: Use autodevops build and register
+    uses: SocialGouv/actions/autodevops-build-register@master
```

Comment:
can be relatif to follow the version of the running action ?

Reply:
```suggestion
    uses: ./autodevops-build-register@master
```

Reply:
Nope, I think I tested it and it didn't work :(